src/integration-test/java/com/mongodb/jdbc/integration/testharness/DataLoader.java

```diff
@@ -94,6 +96,38 @@ public void dropCollections() {
         }
     }
 
+    private void setSchema(String database, String collection, Map<String, Object> jsonSchema) {
+        BsonDocument command = new BsonDocument();
+        BsonDocument schema = new BsonDocument();
+        command.put("sqlSetSchema", new BsonString(collection));
+        command.put("schema", schema);
+        schema.put("version", new BsonInt32(1));
+
+        Document doc = new Document(jsonSchema);
+        schema.put(
+                "jsonSchema",
+                doc.toBsonDocument(BsonDocument.class, MongoClient.getDefaultCodecRegistry()));
+        try (MongoClient mongoClient = new MongoClient(adlUri)) {
+            MongoDatabase db = mongoClient.getDatabase(database);
+            db.runCommand(command);
+        }
+    }
+
+    private void generateSchemaOne(String database, String collection) {
```

Comment:
I'll update the SQL-632 PR to use this generateSchemaOne method instead of generating for the database so that the set schemas don't get overwritten.

Reply:
If that means we won't be using `generateSchema()` anymore, we could refactor this signature to be `generateSchema(String database, String collection)` (basically just a rename).